src/lib/pages/frontsite/Home/Headline/Headline.js

```diff
@@ -51,7 +51,7 @@ const HeadlineLayout = props => {
                     className="headline-quick-access"
                     as={Link}
                     to={FrontSiteRoutes.documentsListWithQuery(
-                      '&sort=mostrecent&order=desc'
+                      '&sort=mostrecent&order=desc&p=1'
```

Comment:
I would probably go for that too, since the issue can still be reproduced if provided a url without the `p=1` parameter; ie. [https://localhost:3000/search?q=](https://localhost:3000/search?q=). Thanks Konstantina for working on it, it's a very annoying issue that has been here for a long time.